base_estimation/ContactStatus[] contacts_status 
